.eslintrc.js

```diff
@@ -44,6 +44,8 @@ module.exports = {
         ],
         "@typescript-eslint/no-misused-promises": "off",
         "@typescript-eslint/no-unsafe-assignment ": "off",
+        // nullish coalescing introduces a lot false negative
+        "@typescript-eslint/prefer-nullish-coalescing": "off",
```

Comment:
```suggestion
    		"@typescript-eslint/prefer-nullish-coalescing": [
          		"error",
     		     {
    		        ignoreConditionalTests: true,
  		        },
```

Have you considered only turning of instances that have a high rate of false positives?

Reply:
Actually a better option would be:

```suggestion
        "@typescript-eslint/prefer-nullish-coalescing": [
        "error",
        {
        	ignorePrimitives: { boolean: true }
        }],
```